src/Magnum/BulletIntegration/DebugDraw.h

```diff
@@ -0,0 +1,177 @@
+            ProfileTimings = DBG_ProfileTimings,        /**< Profile timings */
+            EnableSatComparison = DBG_EnableSatComparison, /**< Enable Sat Comparison */
+            DisableBulletLCP = DBG_DisableBulletLCP,    /**< Disable Bullet LCP */
+            EnableCCD = DBG_EnableCCD,                  /**< Enable CCD */
+            DrawConstraints = DBG_DrawConstraints,      /**< Draw constaints */
+            DrawConstraintLimits = DBG_DrawConstraintLimits, /**< Draw constraint limits */
+            FastWirefram = DBG_FastWireframe,           /**< Draw fast wireframes */
+            DrawNormals = DBG_DrawNormals,              /**< Draw normals */
+            DrawFrames = DBG_DrawFrames,                /**< Draw frames */
+        };
+
+        typedef Containers::EnumSet<Mode> Modes;
+
+        /**
+         * @brief Constructor
+         * @param initialBufferCapacity Amount of lines for which to reserve
+         *        memory in the buffer vector.
+         *
+         * Sets up @ref Shaders::VertexColor3D, @ref Buffer and @ref Mesh for physics debug rendering.
+         */
+        DebugDraw(UnsignedInt initialBufferCapacity=0):
+            btIDebugDraw(),
+            _mesh{MeshPrimitive::Lines}
+        {
+            _mesh.addVertexBuffer(_buffer, 0, Shaders::VertexColor3D::Position{}, Shaders::VertexColor3D::Color{});
+            _bufferData.reserve(initialBufferCapacity*4);
+        }
+
+        /** @brief Set debug mode */
+        virtual void setDebugMode(int debugMode) override {
+            _debugMode = Mode(debugMode);
+        }
+
+        /** @overload */
+        virtual void setDebugMode(Modes debugMode) {
+            _debugMode = debugMode;
+        }
+
+        /** @brief Debug mode */
+        virtual int getDebugMode() const override {
+            return Int(_debugMode);
+        }
+
+        /** @brief Draw a line with one color */
+        virtual void drawLine(const btVector3& from, const btVector3& to, const btVector3& color) override {
+            drawLine(from, to, color, color);
+        }
+
+        /** @brief Draw a line with color gradient. */
+        virtual void drawLine(const btVector3& from, const btVector3& to, const btVector3& fromColor, const btVector3& toColor) override;
+
+        /** @brief Draw contact point */
+        void drawContactPoint(const btVector3& pointOnB, const btVector3& normalOnB, btScalar distance, int CORRADE_UNUSED lifeTime, const btVector3& color) override {
+            drawLine(pointOnB, pointOnB + normalOnB*distance, color);
+        }
+
+        /** @brief Print a warning */
+        virtual void reportErrorWarning(const char *warningString) override {
+            Warning() << "DebugDraw:" << warningString;
+        }
+
+        /** @brief Draw text */
+        virtual void draw3dText(const btVector3& CORRADE_UNUSED location, const char* CORRADE_UNUSED textString) override {
```

Comment:
But some day you may want to implement it and then you would need to look up what the paramters mean? :)